index.html

```diff
@@ -0,0 +1,432 @@
+                        <h2>Illustration</h2>
+                        <p>Graphic / Illustration / poster</p>
+                    </div>
+                </div>
+                <div class="single-portfolio">
+                    <img src="resources/portfolio-img2.jpg" alt="Second ilustration gallery">
+                    <div class="portfolio-text">
+                        <a href="#"><i class="icon-googleplus"></i></a>
+                        <h2>Illustration</h2>
+                        <p>Graphic / Illustration / poster</p>
+                    </div>
+                </div>
+                <div class="single-portfolio">
+                    <img src="resources/portfolio-img3.jpg" alt="Third ilustration gallery">
+                    <div class="portfolio-text">
+                        <a href="#"><i class="icon-googleplus"></i></a>
+                        <h2>Illustration</h2>
+                        <p>Graphic / Illustration / poster</p>
+                    </div>
+                </div>
+                <div class="single-portfolio">
+                    <img src="resources/portfolio-img4.jpg" alt="Quarter ilustration gallery">
+                    <div class="portfolio-text">
+                        <a href="#"><i class="icon-googleplus"></i></a>
+                        <h2>Illustration</h2>
+                        <p>Graphic / Illustration / poster</p>
+                    </div>
+                </div>
+                <div class="single-portfolio">
+                    <img src="resources/portfolio-img5.jpg" alt="Fifth ilustration gallery">
+                    <div class="portfolio-text">
+                        <a href="#"><i class="icon-googleplus"></i></a>
+                        <h2>Illustration</h2>
+                        <p>Graphic / Illustration / poster</p>
+                    </div>
+                </div>
+                <div class="single-portfolio">
+                    <img src="resources/portfolio-img6.jpg" alt="Sixth ilustration gallery">
+                    <div class="portfolio-text">
+                        <a href="#"><i class="icon-googleplus"></i></a>
+                        <h2>Illustration</h2>
+                        <p>Graphic / Illustration / poster</p>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section id="blog-section" class="blog-section">
+            <div class="blog-container container">
+                <div class="blog-top">
+                    <h2 class="section-title">OUR BLOG</h2>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. </p>
+                </div>
+                <div class="blog-content">
+                    <div class="blog-content-items">
+                        <div class="items-content">
+                            <h2>Online Business / dec 9, 2016</h2>
+                            <h3>The best way to make Online Bussines</h3>
```

Comment:
check on the headers, are they suppose to be headers, if so, h2 is not correct as the h2 is Our blog